proftpd/init.sls

```diff
@@ -29,8 +29,16 @@ proftpd:
         - group: root
         - mode: 644
         - template: jinja
+        - context:
```

Comment:
I'm not sure you need to define both `context` and `defaults`, stick with `context` it should be sufficient

Reply:
I confirme, context is enough thanks

Reply:
ok, in reality previously `defaults` was used. I don't think it changes anything but by sevurity maybe it's better to let the previous one in place.
@myii any advice maybe?